cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -77,6 +79,7 @@ public BasicMapReduceContext(Program program,
                                MapReduceSpecification spec,
                                long logicalStartTime,
                                String workflowBatch,
+                               @Nullable String adapterName,
```

Comment:
Fixed. Again, Wanted to keep it next to ``workflowBatch``, but don't feel too strongly.